cmd/switch.go

```diff
@@ -153,8 +153,16 @@ Operational steps to replace a switch:
 		},
 		ValidArgsFunction: c.comp.SwitchListCompletion,
 	}
-
-	return genericcli.NewCmds(cmdsConfig, switchDetailCmd, switchMachinesCmd, switchReplaceCmd, switchSSHCmd, switchConsoleCmd)
+	togglePortCmd := &cobra.Command{
+		Use:   "toggle <switchID> <portID> [<desiredstate>]",
```

Comment:
This interface is quite different from the rest of the commands. Maybe it's a bit more consistent to have `toggle <switch> --port-id port1 --state ON` and completions for the individual flags.